### DISCUSSION TITLE
Tabs Aside for Firefox Nightly Android

### DISCUSSION THREAD

**Discussion by ghost** [tim-we/tabs-aside#128] [Ideas]
I really love this extension and the features which are just right for my usage (reading medical research). It helps with my studies a lot. And it would be awesome if we can get it to work with Firefox for Android. I know extensions are supported with just Nightly on Android but I can't get it to work. Is that feasible?

**Comment by tim-we** [comment]
I haven't looked into that for a while but I will check if its feasible. Feel free to remind me if you have not heard back from me within a week :)

**Reply by ghost** [reply]
Thanks, Tim, it means a lot. I really enjoy using the extension.

**Comment by tim-we** [comment]
Assuming that the information on [this page](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks#browser_compatibility) is up to date it is not possible as extensions cannot create or manage bookmarks. It could be that this has changed in Nightly but I don't know where to get that information. I could create a test extension for that and just try it but I'm currently not that motivated to do this for all the APIs used by Tabs Aside, sorry. If anyone knows more about this please let me know.

**Comment by ghost** [comment]
Not supported is the tab hiding API (`tabs.hide`). And neither is the ability to move tabs to a different position (`tabs.move`). So an extension that remembers tabs, closes tabs and/or opens new tabs. Still, `tabs.query` retrieves the current list of tabs (when the extension is freshly installed). Also, onCreated/onRemoved could be used to track new/closed tabs. 

+At first I was looking to do something with the tab groups in Chrome but still it would take effort. Still, thanks for looking at the option.

**Comment by tim-we** [comment]
If you follow the development of FF on Android (Nightly) feel free to let me know if this changes, happy to publish a version for Android in that case.
